pkg/dispatcher/operator/workers/controller_test.go

```diff
@@ -117,7 +122,8 @@ func TestAssignCalulationDB(t *testing.T) {
 		},
 
 		{
-			id: "multiple values, scrambled existing processing statuses",
+			id:                 "multiple values, scrambled existing processing statuses",
+			redisSortedSetName: "vz",
```

Comment:
Can we also check different values? Also `tc.redisSortedSetName` should be passed to [here](https://github.com/vega-project/ccb-operator/blob/master/pkg/dispatcher/operator/workers/controller_test.go#L155) as well.

Reply:
I tried testing different values of the sorted set name (other than "vz") and it works now.